db/consistency_level.cc

```diff
@@ -237,7 +237,7 @@ filter_for_query(consistency_level cl,
                  const inet_address_vector_replica_set& preferred_endpoints,
                  read_repair_decision read_repair,
                  const gms::gossiper& g,
-                 gms::inet_address* extra,
+                 std::optional<gms::inet_address>* extra,
```

Comment:
This should be a reference really, but this is preexisting problem.

Reply:
No, it should be a pointer, because the call sometimes is passing nullptr there thus saying that it doesn't want to get the extra peer back

Reply:
Ah, ok.